source/includes/aws/_objects.md

```diff
@@ -139,16 +193,26 @@ file: path_to_file
 }
 ```
 
-<code>POST /services/operation/upload</code>
+<code>POST /rest/services/operations/upload</code>
```

Comment:
is this actually the URL for the post ? does it not require the service code and environment name?

Reply:
Yes! It's really weird and maybe should make a technical debt ticket

Reply:
![image](https://user-images.githubusercontent.com/58345485/160483138-7ed6b032-08e6-403b-aefd-20effd9edca8.png)![image](https://user-images.githubusercontent.com/58345485/160483166-a946acea-c43e-41a2-adc4-eadc54091622.png)

Reply:
🤔 weird